### ISSUE TITLE
Doc not quite right

### ISSUE THREAD

**Issue by ShamanTcler** [thclark/cpplot#15]
This is what you have:

A cross-platform compilation file is provided using cmake, it’s not tested on windows but might actually work, since it uses conan to handle all the tricky dependencies.

git clone https://github.com/thclark/cpplot
cd cpplot && cmake install

Running "cmake install" 
```Bash
CMake Error: The source directory "C:/Users/xxxxx/github/cpplot/install" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
```

So then I tried my std. out of source build technique:
```Bash
mkdir build
cd build
cmake .. -G Ninja
```

Got your error ...

So I did a 

```Bash
conan install .. --install-folder C:/Users/xxxxx/github/cpplot/build
```

Got a coffee as Conan churned ...

then another
```Bash
cmake .. -G Ninja
ninja
```

ignoring unknown option '-std=c++11'   ....Hmmm

fatal error C1189: #error:  ERROR macro is defined. Define GLOG_NO_ABBREVIATED_SEVERITIES before including logging.h. See the document for detail.

So kinda sorta worked ... looking into the error

**Comment by ShamanTcler** [comment]
Ok CMakeLists.txt suggestion

replace:
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")

with
set (CMAKE_CXX_STANDARD 11)

This is cross platform "clean"

**Comment by ShamanTcler** [comment]
Seems Google log and windows don't play nice ... try adding:

IF(CMAKE_SYSTEM_NAME STREQUAL Windows)
    add_definitions(-DGLOG_NO_ABBREVIATED_SEVERITIES)
endif()

to the CMake

**Comment by ShamanTcler** [comment]
Hmmm 
```Bash
main.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl google::InitGoogleLogging(char const *)" (__imp_?InitGoogleLogging@google@@YAXPEBD@Z) referenced in function main
```

But in the link line I see:
```Bash 
-LIBPATH:C:\Users\xxxxxx\.conan\data\glog\0.4.0\_\_\package\57524c5f081b8161c3d2b758fe2c7b1f3123a7d8\lib
glog
``` 

Now from conan I see this is version 0.4.0 is this the correct one?

**Comment by thclark** [comment]
> Now from conan I see this is version 0.4.0 is this the correct one?
I'm using 0.4.0 here (OSX)...

**Comment by thclark** [comment]
I cleared up some system dependent stuff for glog when moving to conan.
Try some of the older system dependent stuff:

https://github.com/thclark/cpplot/blob/ef0186e4d2b6fba09a34417a9b45287ee3b27e27/source/CMakeLists.txt#L24

**Comment by thclark** [comment]
I believe this was closed by #16 and #19
